invenio_accounts/ext.py

```diff
@@ -197,6 +210,18 @@ def delay_security_email(msg):
         self.kvsession_extension = KVSessionExtension(
             session_kvstore, app)
 
+        # Warn if inconsistent configuration is detected
+        sec = app.extensions["security"]
+        local_login = app.config.get("ACCOUNTS_LOCAL_LOGIN_ENABLED", True)
+        x_able = sec.registerable or sec.changeable or sec.recoverable
+        if x_able and not local_login:
+            warning_message = (
+                "ACCOUNTS_LOCAL_LOGIN_ENABLED is False, while at least one "
+                "of SECURITY_REGISTERABLE, SECURITY_REGISTERABLE and "
+                "SECURITY_REGISTERABLE is True"
+            )
+            warn(warning_message, Warning, stacklevel=2)
```

Comment:
i figured, a warning would be nicer than an exception because there *might* be use cases for this?
stacklevel=2 makes the display of the warning nicer (it'll show as `init_app()`, which is nicer than `warn(...)`